kornia/geometry/dsnt.py

```diff
@@ -12,6 +12,7 @@
 from kornia.utils.grid import create_meshgrid
 
 
+@torch.jit.ignore
 def _validate_batched_image_tensor_input(tensor):
     if not torch.is_tensor(tensor):
         raise TypeError("Input type is not a torch.Tensor. Got {}"
```

Comment:
you can use `isinstance(tensor, torch.Tensor)` instead. Actually would be great if we can stat using here a generic API for checkings. In this case `check_is_tensor`.

See discussion: https://github.com/kornia/kornia/pull/458#discussion_r417838712

if you go ahead with this, place it under `kornia.testing`

Reply:
Ah yes, `isinstance` does work---that's good to know. I'm not quite sure how you want these `check_*` functions to work though. Should they return a bool, perform an assert, or throw an informative exception if the check fails (eg TypeError)?

Reply:
Was thinking in raising a TypeError in this case